### PR TITLE
CIR-1821 - Make validation changes

### PR DESCRIPTION
**Pull request by burns238**

### CODE REVIEW

File: app/v1/validation/fullReturn/AdjustedGroupInterestValidator.scala [hunk 1]
```diff
@@ -57,34 +56,21 @@ trait AdjustedGroupInterestValidator extends BaseValidation {
...
-      case (_, groupEBITDA, groupRatio) if groupEBITDA <= 0 && groupRatio == 100 =>
-        adjustedGroupInterestModel.groupRatio.validNec
-      case (_, groupEBITDA, groupRatio) if groupEBITDA <= 0 && groupRatio != 100 =>
+      case (_, Some(groupEBITDA), groupRatio) if groupEBITDA <= 0 && groupRatio != 100 =>
         NegativeOrZeroGroupEBITDAError(groupRatio).invalidNec
       case _ => adjustedGroupInterestModel.groupRatio.validNec
     }
   }
 
-  private def groupRatioCalculationDoesntMatch(qngie: BigDecimal, groupEBITDA: BigDecimal, groupRatio: BigDecimal): Boolean = {
```
Comment (AdamCoops) [inline app/v1/validation/fullReturn/AdjustedGroupInterestValidator.scala:78]: Has this been deleted on purpose?

Comment (burns238) [reply app/v1/validation/fullReturn/AdjustedGroupInterestValidator.scala:78]: Yeah, it wasn't used. We removed its use in a previous development but we mustn't have removed the method itself

File: test/v1/services/NrsServiceSpec.scala [hunk 2]
```diff
@@ -32,6 +31,8 @@ import v1.services.mocks.MockNrsConnector
 import java.nio.charset.StandardCharsets.UTF_8
 import java.util.UUID
 import scala.concurrent.Future
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AsyncWordSpec
 
 class NrsServiceSpec extends AsyncWordSpec with MockNrsConnector with Matchers {
```
Comment (AdamCoops) [inline test/v1/services/NrsServiceSpec.scala:37]: This is AsyncWordSpec rather than AnyWordSpec like all the others, is this deliberate?

**Review by AdamCoops — APPROVED** [review]

File: test/v1/services/NrsServiceSpec.scala (hunk 2 above)
Comment (burns238) [reply test/v1/services/NrsServiceSpec.scala:37]: Yeah it was already AsyncWordSpec, this just changed where it was getting imported from

**Pull request merged**